app/templates/components/public/side-menu.hbs

```diff
@@ -1,80 +1,80 @@
 {{#if (eq this.session.currentRouteName 'public.index')}}
   <a href='#info' {{action "scrollToTarget" 'info' }} class='item active scroll'>
-    <div><i class="{{unless this.device.isMobile 'hidden-item'}} info icon"></i>{{t 'Info'}}</div>
+    <div><i class="mr-2 {{unless this.device.isMobile 'hidden-item mr-2'}} info icon"></i>{{t 'Info'}}</div>
```

Comment:
why two times?

Reply:
Already corrected, see the commit above your review!